tests/test_handler_checkpoint_saver.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import tempfile
+import shutil
+import torch
+import unittest
+from ignite.engine import Engine
+from monai.handlers import CheckpointSaver
+from parameterized import parameterized
+import logging
+import sys
+
+TEST_CASE_1 = [True, False, None, 1, 0, None, 0, None, ["test_net_final_iteration=40.pth"]]
```

Comment:
would be great to have a unit test on multigpu model loading and saving...

Reply:
added new tests in the latest patch.
Thanks.